app/helpers/icons_helper.rb

```diff
@@ -152,4 +161,56 @@ def github_icon(css_classes = "")
     SVG
     .html_safe
   end
+
+  def hamburger_icon(css_classes = "")
+    <<~SVG
+      <svg class="#{css_classes}" width="20" height="20" viewBox="0 0 20 20" fill="none" stroke="currentColor" xmlns="http://www.w3.org/2000/svg">
+        <path d="M10 3C10.8284 3 11.5 3.67157 11.5 4.5C11.5 5.32843 10.8284 6 10 6C9.17157 6 8.5 5.32843 8.5 4.5C8.5 3.67157 9.17157 3 10 3Z" fill="#0F172A"/>
+        <path d="M10 8.5C10.8284 8.5 11.5 9.17157 11.5 10C11.5 10.8284 10.8284 11.5 10 11.5C9.17157 11.5 8.5 10.8284 8.5 10C8.5 9.17157 9.17157 8.5 10 8.5Z" fill="#0F172A"/>
+        <path d="M11.5 15.5C11.5 14.6716 10.8284 14 10 14C9.17157 14 8.5 14.6716 8.5 15.5C8.5 16.3284 9.17157 17 10 17C10.8284 17 11.5 16.3284 11.5 15.5Z" fill="#0F172A"/>
+      </svg>
+    SVG
+    .html_safe
+  end
+
+  def evaluation_created_icon(css_classes = "")
```

Comment:
I propose to use a more straightforward name:

```suggestion
  def clock_icon(css_classes = "")
```